theme/saas/static/css/shippable.css

```diff
@@ -142,3 +142,284 @@ blockquote {
   color: #333;
   border-left: 5px solid #ddd;
 }
+
+.containerpad {
+  padding-left: 20px;
+  padding-right: 40px;
+}
+
+.content-pad {
+  padding: 2rem;
+}
+.navbar-default.navbar-transparent .navbar-brand img {
+  bottom: 15px !important;
+}
+body,html,.row-offcanvas {
+  height:100%;
+}
+
+body {
+  padding-top: 50px;
+}
+
+menu,ol,small,ul {
+    margin: 0;
+    padding: 0
+}
+
+menu,ol,ul {
+    list-style: none
+}
+
+#sidebar {
+  width: inherit;
+  min-width: 220px;
+  background-color:#f5f5f5;
+  float: left;
+  height:100%;
+  position:relative;
+  overflow-y:auto;
+  overflow-x:hidden;
+}
+
+/* ------------------ */
+.shipContent {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: stretch;
+    align-items: stretch;
+    -ms-flex-direction: row-reverse;
+    flex-direction: row-reverse;
+    height: calc(100vh - 63px)
+}
+
+.shipContent,.shipContent-body {
+    overflow-y: auto;
+    overflow-x: hidden
+}
+
+.shipContent-body {
+    -ms-flex: 1;
+    flex: 1
+}
+
+.shipContent-sidebar {
+    width: 300px;
+    padding: 3rem;
+    background: #fafbfc;
+    border-right: 1px solid #e4edf5;
+    overflow-y: auto;
+    overflow-x: hidden
+}
+
+.shipDocsNav {
+    position: relative;
+    text-align: left;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: center;
+    justify-content: center
+}
+
+.shipDocsNav-tab {
+    display: inline-block;
+    font-size: 16px;
```

Comment:
I think font sizes should be in relative units. Using pixels will not allow things to scale uniformly if we change the base font sizes. We need to be able to maintain the ratios across the board if we want to change the sizes later.